common/app/routes/challenges/Test-Suite.jsx

```diff
@@ -39,6 +47,24 @@ export default class TestSuite extends PureComponent {
     });
   }
 
+  getAccessibleText(err, pass, text) {
```

Comment:
Why does this function need to be a method attached to the class? It should just be a top level function in this file